nftables_test.go

```diff
@@ -173,6 +175,52 @@ func TestList(t *testing.T) {
 	}
 }
 
+func TestListFlowtables(t *testing.T) {
+	for _, tc := range []struct {
+		name       string
+		objType    string
+		nftOutput  string
+		listOutput []string
+	}{
+		{
+			name:       "empty list",
+			objType:    "flowtables",
+			nftOutput:  `{"nftables": [{"metainfo": {"version": "1.0.1", "release_name": "Fearless Fosdick #3", "json_schema_version": 1}}]}`,
+			listOutput: nil,
+		},
+		{
+			name:       "singular objType",
+			objType:    "flowtables",
+			nftOutput:  `{"nftables": [{"metainfo": {"version": "1.0.6", "release_name": "Lester Gooch #5", "json_schema_version": 1}}, {"flowtable": {"family": "inet", "name": "test3", "table": "kindnet-ipmasq", "handle": 3273, "hook": "ingress", "prio": 5, "dev": "eth0"}}, {"flowtable": {"family": "inet", "name": "test", "table": "kindnet-dnscache", "handle": 245503, "hook": "ingress", "prio": 0}}, {"flowtable": {"family": "inet", "name": "test2", "table": "kindnet-dnscache", "handle": 2859641, "hook": "ingress", "prio": 0, "dev": "eth0"}}]}`,
+			listOutput: nil,
+		},
+		{
+			name:       "plural objType",
+			objType:    "flowtables",
+			nftOutput:  `{"nftables": [{"metainfo": {"version": "1.0.6", "release_name": "Lester Gooch #5", "json_schema_version": 1}}, {"flowtable": {"family": "inet", "name": "test3", "table": "kindnet-ipmasq", "handle": 3273, "hook": "ingress", "prio": 5, "dev": "eth0"}}, {"flowtable": {"family": "inet", "name": "test", "table": "kindnet-dnscache", "handle": 245503, "hook": "ingress", "prio": 0}}, {"flowtable": {"family": "inet", "name": "test2", "table": "kindnet-dnscache", "handle": 2859641, "hook": "ingress", "prio": 0, "dev": "eth0"}}]}`,
+			listOutput: nil,
```

Comment:
You have `listOutput: nil` in all of these cases, suggesting it is not actually parsing the output as you expected.

Also, you could just add a flowtable test to `TestList`

Reply:
I took Chains as reference and replicated it, I thought the same, this test is not adding much value

Reply:
TestList has
```
					args:   []string{"/nft", "--json", "list", "chains", "ip"},

```

that couples the testscases, that is why I duplicated it

I will remove it as there is no much use cases for the List